### PR TITLE
Add E2E scenarios (NR-52126)

### PR DESCRIPTION
**Pull request by carlossscastro**
Adding 3 E2E test scenarios with different mongo versions:

Sharded cluster (mongos) on mongo 4.2
Replicaset on mongo 5
Standalone on mongo 6

### CODE REVIEW

File: exporters/mongodb/e2e/docker-compose-standalone.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    image: mongo:6.0
+    depends_on:
+      - "e2e_standalone"
+    links:
+      - e2e_standalone:e2e_standalone
+    volumes:
+      - ./scripts:/scripts
+    environment:
+      - MONGO1=e2e_standalone
+    entrypoint: [ "/scripts/setup-standalone.sh" ]
```
Comment (sigilioso) [inline exporters/mongodb/e2e/docker-compose-standalone.yml:24]: nit: missing EOF

**Review by sigilioso — APPROVED** [review]
LGTM 👍 

There is something which has just come to my mind (totally out of the scope of this PR): It would be really useful to run the e2e in pipelines before merging changes, even if we needed to trigger it manually.

File: exporters/mongodb/e2e/docker-compose-standalone.yml (hunk 1 above)
Comment (carlossscastro) [reply exporters/mongodb/e2e/docker-compose-standalone.yml]: 👍

**Comment by carlossscastro** [comment]
> It would be really useful to run the e2e in pipelines before merging changes, even if we needed to trigger it manually

Definitely a good idea. Creating a task for it

**Pull request merged**